platform/security/platform-security-core-api/src/main/java/ddf/security/liberty/paos/Request.java

```diff
@@ -0,0 +1,85 @@
+/**
+ * Copyright (c) Codice Foundation
+ * <p>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ **/
+package ddf.security.liberty.paos;
+
+import javax.xml.namespace.QName;
+
+import org.opensaml.saml.common.SAMLObject;
+import org.opensaml.saml.common.xml.SAMLConstants;
+import org.opensaml.soap.soap11.ActorBearing;
+import org.opensaml.soap.soap11.MustUnderstandBearing;
+
+public interface Request extends SAMLObject, MustUnderstandBearing, ActorBearing {
+
+    String DEFAULT_ELEMENT_LOCAL_NAME = "Request";
+
+    QName DEFAULT_ELEMENT_NAME = new QName(SAMLConstants.PAOS_NS,
+            DEFAULT_ELEMENT_LOCAL_NAME, SAMLConstants.PAOS_PREFIX);
+
+    String TYPE_LOCAL_NAME = "RequestType";
+
+    QName TYPE_NAME = new QName(SAMLConstants.PAOS_NS, TYPE_LOCAL_NAME,
```

Comment:
Going to leave this just because all of the other Opensaml objects have it as well to be consistent.